### DISCUSSION TITLE
Scale up : Epiphany Cluster

### DISCUSSION THREAD

**Discussion by diwa2508** [epiphany-platform/epiphany#2760] [Q&A]
Hi,

Is there a provision to scale up or scale down the number of nodes in the Kubernetes cluster through epiphany  //?

**Comment by rafzei** [comment]
Hello @diwa2508,
Please check our official documentation here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/howto/CLUSTER.md#how-to-scale-or-cluster-components

Regards, Rafal.
